ui/article-renderer/src/main/java/org/cru/godtools/article/fragment/ArticlesFragment.java

```diff
@@ -0,0 +1,95 @@
+package org.cru.godtools.article.fragment;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.cru.godtools.article.R;
+import org.cru.godtools.article.R2;
+import org.cru.godtools.article.adapter.ArticleAdapter;
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.view_model.ArticleViewModel;
+import org.cru.godtools.base.tool.fragment.BaseToolFragment;
+
+import java.util.Locale;
+import java.util.Objects;
+
+import butterknife.BindView;
+import timber.log.Timber;
+
+public class ArticlesFragment extends BaseToolFragment implements ArticleAdapter.Callback {
+    public static final String TAG = "ArticlesFragment";
+    private static final String MANIFEST_KEY = "manifest-key";
+
+    @Nullable
+    @BindView(R2.id.articles_recycler_view)
+    RecyclerView mArticlesView;
+
+    ArticleAdapter mAdapter;
+
+    public static ArticlesFragment newInstance(@NonNull final String code, @NonNull final Locale locale,
+                                               String manifestKey) {
+        ArticlesFragment fragment = new ArticlesFragment();
+        Bundle args = new Bundle();
+        populateArgs(args, code, locale);
+        args.putString(MANIFEST_KEY, manifestKey);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        super.onCreateView(inflater, container, savedInstanceState);
+        return inflater.inflate(R.layout.fragment_articles, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        setupArticleRecyclerView();
+    }
+
+    private void setupArticleRecyclerView() {
+        if (mArticlesView != null) {
+            mAdapter = new ArticleAdapter();
+            mAdapter.setCallbacks(this);
+            mAdapter.setToolManifest(mManifest);
+            DividerItemDecoration itemDecoration = new DividerItemDecoration(getContext(),
+                    DividerItemDecoration.VERTICAL);
+            itemDecoration.setDrawable(Objects.requireNonNull(getActivity()).getResources()
+                    .getDrawable(R.drawable.divider));
+
+            mArticlesView.addItemDecoration(itemDecoration);
+            mArticlesView.setLayoutManager(new LinearLayoutManager(getActivity()));
+            mArticlesView.setAdapter(mAdapter);
+
+            ArticleViewModel viewModel = ArticleViewModel.getInstance(getActivity());
```

Comment:
I think I'm going to create a commit on your branch with what I'm thinking of for the fragment view model